tests/tailer/savedown_withcount/tests.csv

```diff
@@ -0,0 +1,16 @@
+action,ms,bytes,lang,code,repeate,minver,comment
+before,0,0,q,tr:.servers.gethandlebytype[`tr_seg1;`any],1,,"Connect to tailreader handle to give access the tailreader1 for testing tailer1"
+before,0,0,q,tl:.servers.gethandlebytype[`tailer_seg1;`any],1,,"Connect to tailer1 handle,so tailer can be tested through tl handle"
+before,0,0,q,tl".wdb.numrows:100",1,,"Set row count as the standard row count is 10000 would be to large for a test"
+run,0,0,q,(tr"count tables[]")=2,1,,"Check the number of tables in the tailreader is currently 2"
+run,0,0,q,tl".ds.savealltablesoverperiod[]",1,,"Running savedown function sould save tables but, there should be no data to savedown so the table count should stay the same"
+run,0,0,q,(tr"count tables[]")=2,1,,"Check the number of tables in the tailreader has remained the same indicating the savedown didn't add any tables as there was none to add"
+run,0,0,q,.os.sleep 10,1,,"Wait 10 seconds to populate tables with rows of data for trade and quote"
+run,0,0,q,tl"![;();0b;`symbol$()]`trade";tl".ds.savealltablesoverperiod[]",1,,"Running savedown function on tables,should save quote"
```

Comment:
Why the functional form here? Surely we can just do this?

```suggestion
run,0,0,q,tl"delete from `trade";tl".ds.savealltablesoverperiod[]",1,,"Running savedown function on tables,should save quote"
```

it's also not explained in the comment why we're deleting from trade table